src/scss/themes/_hero.scss

```
@@ -159,6 +159,11 @@
 		text-align: center;
 	}
 
+	.o-teaser__meta
```

These suggestions move the opening brace to fix the linting issue: 
```suggestion
	.o-teaser__meta {
```